apis/config/v1alpha2/configuration_types.go

```diff
@@ -51,6 +51,10 @@ type Configuration struct {
 	// This is achieved by blocking the start of new jobs until the previously
 	// started job has all pods running (ready).
 	WaitForPodsReady *WaitForPodsReady `json:"waitForPodsReady,omitempty"`
+
+	// Client provides additional configuration options for Kubernetes
+	// API server client.
+	Client *Client `json:"client,omitempty"`
```

Comment:
ClientConnection, to match kube-scheduler

Reply:
ComponentConfig has couple more fields that would require a slightly bigger refactoring to get them supported (like dropping kubefile flag etc). Not a big deal but not sure if it is actually needed.

Reply:
As long as we stay consistent, we can swap the types in the future.

Reply:
Done.